src/test/resources/catalinazoldi/Mobile.xml

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
+
+<suite name="Desktop" verbose="1" >
+    <test name="Test desktop" >
+        <groups>
+            <run>
+                <exclude name="Desktop"/>
```

Comment:
"desktop" instead of "Desktop". That's how you named the groups in test classes.